libcxx/utils/libcxx/header_information.py

```diff
@@ -56,6 +56,7 @@
     "cwchar": "// UNSUPPORTED: no-wide-characters",
     "cwctype": "// UNSUPPORTED: no-wide-characters",
     "experimental/iterator": "// UNSUPPORTED: c++03",
+    "experimental/math": "// UNSUPPORTED: c++03, c++11, c++14",
```

Comment:
Is this a leftover?

Reply:
Yes. Fixed in fc667d6e28dc4b4ec139d42cc36bb2b1414f879a.